ledger/metrics/src/test/scala/com/daml/metrics/MetricsSpec.scala

```diff
@@ -14,8 +14,7 @@ class MetricsSpec extends AsyncWordSpec with Matchers {
       val metrics = new Metrics(new MetricRegistry)
```

Comment:
As the `registerGauge` function has been put into the package object you might want to have a single test suite for that instead of having two separate test suites testing the same function (i.e., these are redundant tests).

Reply:
Good call. Done!